pkg/antctl/raw/supportbundle/command.go

```diff
@@ -403,7 +409,7 @@ func createControllerClient(k8sClientset kubernetes.Interface, antreaClientset a
 	cfg.Host = net.JoinHostPort(nodeIP, fmt.Sprint(controllerInfo.APIPort))
 	controllerClient, err := rest.RESTClientFor(cfg)
 	if err != nil {
-		klog.Warningf("Error when creating controller client for node: %s", controllerInfo.NodeRef.Name)
+		klog.InfoS("Error when creating controller client for node: %s", controllerInfo.NodeRef.Name)
```

Comment:
```suggestion
		klog.InfoS("Error when creating controller client for Node", "nodeName", controllerInfo.NodeRef.Name)
```

Structured logging does not use formatters.